README.md

```diff
@@ -51,17 +51,19 @@ Once the build is done, your app image is available in
 Available Demos
 ---------------
 
-The default login credentials for all demos are `root` without any
+The default login credentials for all demo apps are `root` without any
 password.
 
-| **README**     | **Nightly Image**  | **Description**                                  |
-|----------------|--------------------|--------------------------------------------------|
-| [Backbone][]   | [backbone.zip][]   | Link aggregate (lag) and daisy-chain backbone    |
-| [DHCP Boot][]  | [dhcp-boot.zip][]  | Convert .conf and net boot WeOS from a container |
+| **Architecture** | **Platform Code Name** | **Backbone App** | **Dhcp-boot App** |
+|------------------|------------------------|-----------------------|----------------------|
+| powerpc          | Coronet                | [backbone-coronet][]       | [dhcp-boot-coronet][]       |
+| arm cortex-a9    | Dagger                 | [backbone-dagger][]        | [dhcp-boot-dagger][]                  |
+| x86_64           | Zero                   | [backbone-zero][]          | [dhcp-boot-zero][]          |
 
-> **Note:** Currently the builds and defconfigs available here target
->           the Westermo _Coronet_ platform.  Other platforms planned to
->           be supported are the _Dagger_ and _Zero_ platforms.  For
+
+> **Note:** Currently the nightly builds and defconfigs available here target
+>           the Westermo _Coronet_, _Dagger_ and _Zero_ platforms.  Other platforms planned to will 
```

Comment:
Itty bitty comment here on line lengths in text files,  I try to keep them <80 chars.  I have Emacs set up to reflow paragraphs for me with Meta-Q, so I have it easy.  The intent is to make it easy on the eyes also when reading text files using less or similar on a small terminal.